lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :sum, :basket, :menu
+
+  def initialize(order_menu)
+    @menu = order_menu.list
```

Comment:
It feels like it's breaking encapsulation a bit to have order reach into `order_menu` to grab `list`.  Could `Menu` have some methods on it that examine `list` to answer questions like "is this item on the menu?"